### PR TITLE
nawal

### PR DESCRIPTION
**Pull request by nawaldee21**
#### Your Name:

---

### CODE REVIEW

**Comment by nawaldee21** [comment]
fix it

File: relaxr.html/relaxr.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        <button>Sign Up Now!</button>
+                    </div>
+                </div>
+            
+            </body>
+            
+            <footer>
+                <h2>Relaxr</h2>
+                <div>
+                <img src="images/twitter.png" alt="">
```
Comment (remharbi) [inline relaxr.html/relaxr.html]: the link should be '../starter_code/images/twitter.png'

File: relaxr.html/relaxr.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                <img src="images/facebook.png" alt="">
```
Comment (remharbi) [inline relaxr.html/relaxr.html]: the link should be '../starter_code/images/facebook.png'

File: relaxr.html/style.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+}
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+body {
+    color: #000;
+    font-size: 12px;
+    line-height: 1.4;
+    font-family: Helvetica, Arial, sans-serif;
+}
+.re{
+    background-image: url("../images/header_bg.jpg");
```
Comment (remharbi) [inline relaxr.html/style.css]: the link should be '../starter_code/images/header_bg.jpg'

File: relaxr.html/style.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+
+.hrr li{
+    margin: 25px;
+    font-size: 18px;
+list-style: circle;
+}
+.wee {
+    background-image: url('../images/testimonial_bg.jpg');
```
Comment (remharbi) [inline relaxr.html/style.css]: the link should be '../starter_code/images/testimonial_bg.png'

**Review by remharbi — COMMENTED** [review]
Great work Nawal! pay more attention to links

**Pull request closed without merging**